naslib/search_spaces/simple_cell/graph.py

```diff
@@ -6,7 +6,7 @@
 from naslib.search_spaces.core.graph import Graph, EdgeData
 from naslib.search_spaces.core import primitives as ops
 
-from ..nasbench301.graph import _truncate_input_edges
+# from ..nasbench301.graph import _truncate_input_edges
```

Comment:
Can the unused input be removed?

Reply:
_truncate_input_edge is not called or used in Simple cell that's why I removed it. I have put it back in the new commit.